docs/user/ppl/cmd/describe.rst

```diff
@@ -0,0 +1,48 @@
+=============
+describe
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| Using ``describe`` command to query metadata of the index. ``describe`` command could be only used as the first command in the PPL query.
+
+
+Syntax
+============
+describe <index>
+
+* index: mandatory. describe command must specify which index to query from.
+
+
+Example
+=======
+
+The example describes accounts index.
```

Comment:
add another example to demo combination of where or fields command

Reply:
done